docs/core_docs/src/theme/Feedback.js

```diff
@@ -161,6 +164,33 @@ export default function Feedback() {
     setFeedbackSent(true);
   };
 
+  const handleFeedbackDetails = async (e) => {
+    e.preventDefault();
+    if (!feedbackId) {
+      setFeedbackDetailsSent(true);
+      return;
+    }
+    const details = e.target.elements
+      .namedItem("details")
+      ?.value.slice(0, 1024);
+    if (!details) {
+      return;
+    }
+    const supabase = createClient(
+      siteConfig.customFields.supabaseUrl,
+      siteConfig.customFields.supabasePublicKey
+    );
+    const { error } = await supabase.from("feedback_details").insert({
+      feedback_id: feedbackId,
+      details,
+    });
+    if (error) {
+      console.error("Failed to add feedback details", error);
+      throw error;
```

Comment:
Do we want to actually throw this error in production? I think it would be better to just silently fail.

Reply:
Adding on that, I'd wrap this function in a try/catch so if something else goes wrong it doesn't throw, and instead just silently logs the error and returns.

Reply:
It'll just show up in the console which I think is fine, probably don't need both though